src/app/ui/dashboard/sidenav.tsx

```diff
@@ -1,5 +1,5 @@
-import ZusflixLogo from "@/app/ui/zusflix-logo";
-import NavLinks from "@/app/ui/dashboard/nav-links";
+import ZusflixLogo from "@/app/ui/ZusflixLogo";
+import NavLinks from "@/app/ui/dashboard/NavLinks";
```

Comment:
JSX/React (.tsx/.jsx) component must be always named in so called "pascal case" way like this

Reply:
Done.